cmake/OpenCVFindLibsGUI.cmake

```diff
@@ -82,6 +82,7 @@ if(WITH_OPENGL)
       else()
         ocv_include_directories(${OPENGL_INCLUDE_DIR})
       endif()
+      ocv_check_modules(GLEW glew)
```

Comment:
1. This doesn't work on native Win32 (utilizes `pkg-config`)
2. Use CMake's way instead: https://cmake.org/cmake/help/v3.0/module/FindGLEW.html

Reply:
FindGLEW doesn't work on Linux -- it finds only GLEW library; pkg-config returns all necessary flags:
```
$ pkg-config --libs glew
-lGLEW -lX11 -lGLU -lGL
```